### ISSUE TITLE
[Documentation] Proposal: Drop rule SA1609

### ISSUE THREAD

**Issue by persn** [LateStartStudio/Hero6#131]
Warning
SA1609
CSharp.Documentation : The public or protected property's documentation header must contain a value tag.

I don't care for this rule really, it's so tedious, the value tag rarely adds any new substantial information, and in most cases it just increases the verbosity of our code files. It ends up being a repeat of the summary tag.

I propose we just remove this from our rule sets.

```
// Which is better?

/// <summary>
/// Gets the id of the walk area, represented by a pixel of color, using RGBA values as
/// identifiers.
/// </summary>
public Color Id { get; }

/// <summary>
/// Gets the id of the walk area, represented by a pixel of color, using RGBA values as
/// identifiers.
/// </summary>
/// <value>
/// The id of the walk area, represented by a pixel of color, using RGBA values as
/// identifiers.
/// </value>
public Color Id { get; }
```

**Comment by persn** [comment]
@robertkety penny for your thoughts...

**Comment by persn** [comment]
I'm just going to veto this, I want it included in Pull Request #125
